### PR TITLE
Combobox free text docs to site

### PR DESCRIPTION
**Pull request by ThusharaJ07**

### CODE REVIEW

File: site/src/examples/combo-box/FreeText.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+} from "react";
+import { shortColorData } from "./exampleData";
+
+export const FreeText = (): ReactElement => {
+  const [value, setValue] = useState("");
+  const [selectedValues, setSelectedValues] = useState<string[]>([]);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    // React 16 backwards compatibility
+    event.persist();
```
Comment (joshwooding) [inline site/src/examples/combo-box/FreeText.tsx:16]:
```suggestion
```

File: site/src/examples/combo-box/FreeText.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    const value = event.target.value;
+    setValue(value);
+  };
+
+  const handleSelectionChange = (
+    event: SyntheticEvent,
+    newSelected: string[],
+  ) => {
+    // React 16 backwards compatibility
+    event.persist();
```
Comment (joshwooding) [inline site/src/examples/combo-box/FreeText.tsx:26]:
```suggestion
```

File: site/src/examples/combo-box/FreeText.tsx (hunk 1 above)
Comment (ThusharaJ07) [reply site/src/examples/combo-box/FreeText.tsx]: removed

File: site/src/examples/combo-box/FreeText.tsx (hunk 2 above)
Comment (ThusharaJ07) [reply site/src/examples/combo-box/FreeText.tsx]: removed

File: site/docs/components/combo-box/examples.mdx [hunk 3]
```diff
@@ -201,5 +201,14 @@ To style `ComboBox` with a full border, set `bordered={true}`.
 
 We recommend this styling when the field uses the same fill color as the background (i.e., a primary fill color on a primary background).
 
+</LivePreview>
+<LivePreview componentName="combo-box" exampleName="FreeText">
+
+## Free Text
+
+"Free text" is a feature in an input field that allows users to enter text freely, even if it doesn’t match predefined options.
```
Comment (joshwooding) [inline site/docs/components/combo-box/examples.mdx:209]:
```suggestion
If users need to select an option not in a predefined list, use a controlled `ComboBox` and add an `Option` to the bottom of the list matching the typed-in text.
```

**Review by joshwooding — APPROVED** [review]

**Review by shey-v — APPROVED** [review]

**Pull request merged**